scripts/management/MgmtDB.py

```diff
@@ -276,18 +276,24 @@ def get_runnable_tasks(
 
         return runnable_tasks
 
-    def is_task_complete(self, task):
-        process, run_name, status = task
+
+    def num_task_complete(self, task, like=False):
```

Comment:
Is the like argument being added for a reason?

Reply:
like was added to be able to support LIKE operator instead of = and query `run_name LIKE "faultname_REL%"`